### PR TITLE
Upgrade to .Net 6

### PR DESCRIPTION
**Pull request by DavidNinerAtNewDay**
This PR Upgrades to .Net 6 and is essential for the upgrade of Unified API to .Net 6 as without this upgrade the Open API generation fails.

### CODE REVIEW

**Review by kunalgujar — DISMISSED** [review]
Approved.

File: test/FunctionsValidationFilter.Tests/FunctionsValidationFilter.Tests.csproj [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+    <TargetFramework>net6</TargetFramework>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
-    <PackageReference Include="Moq" Version="4.18.2" />
-    <PackageReference Include="xunit" Version="2.4.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.5.0-preview-20221003-04" />
```
[the remark this answers is not in the document]
Comment (DavidNinerAtNewDay) [reply test/FunctionsValidationFilter.Tests/FunctionsValidationFilter.Tests.csproj:9]: I am not sure why that is there but it is in the original main and so probably outside the scope of this change.

**Review by julijamaliseva — APPROVED** [review]

File: test/FunctionsValidationFilter.Tests/FunctionsValidationFilter.Tests.csproj (hunk 1 above)
[the remark this answers is not in the document]
Comment (safakgur) [reply test/FunctionsValidationFilter.Tests/FunctionsValidationFilter.Tests.csproj:9]: I don't see it in `main`:

image

**Comment by robcthegeek** [comment]
Let's get the code warnings removed - then will approve so we can ship this thing! 🚢💨

**Review by robcthegeek — APPROVED** [review]
Appreciate the fixes, and the contribution! **Thank you!**

**Pull request merged**